### PR TITLE
Fix backend vercel deployment

### PR DESCRIPTION
**Pull request by cdcyang**
Vercel forces the backend to be serverless and seems not allowing long running sessions, made change accordingly to handle it.

### CODE REVIEW

**Comment by cdcyang** [comment]
@datu925 I still don't have permission to view/deploy the application to vercel, can you please add the "backend" deployment as part of git build check? There is a guide here https://vercel.com/docs/monorepos

**Comment by datu925** [comment]
> @datu925 I still don't have permission to view/deploy the application to vercel, can you please add the "backend" deployment as part of git build check? There is a guide here https://vercel.com/docs/monorepos

Done. I had to create a new Pro account so you could get access and transfer the projects over, so you might need to re-request permissions, but after that you should be fine.

File: backend/src/server.ts
Comment (datu925) [inline backend/src/server.ts:1]: Please remove the commented out code and the interfaces that look unused at this point.

**Review by datu925 — APPROVED** [review]
Looks fine; see my comment before merging.

**Pull request merged**